virtual_rainforest/models/abiotic_simple/__init__.py

```diff
@@ -0,0 +1,31 @@
+r"""The :mod:`~virtual_rainforest.models.abiotic_simple` module is one of the component
+models of the Virtual Rainforest. It is comprised of several submodules that calculate
+the microclimate and hydrology for the Virtual Rainforest.
+
+Each of the abiotic sub-modules has its own API reference page:
+
+* The :mod:`~virtual_rainforest.models.abiotic_simple.abiotic_simple_model` submodule
+  instantiates the AbioticSimpleModel class which consolidates the functionality of the
+  abiotic simple module into a single class, which the high level functions of the
+  Virtual Rainforest can then use.
+
+* The virtual_rainforest.models.abiotic_simple.simple_regression submodule
```

Comment:
Maybe stick a TODO in somewhere around here to remind yourself to make it a link once that submodule is defined? It's the sort of thing that I'm always convinced I will remember but always forget